src/diffusers/loaders/peft.py

```diff
@@ -54,6 +54,7 @@
     "SanaTransformer2DModel": lambda model_cls, weights: weights,
     "Lumina2Transformer2DModel": lambda model_cls, weights: weights,
 }
+_NO_CONFIG_UPDATE_KEYS = ["to_k", "to_q", "to_v"]
```

Comment:
```suggestion
```

Reply:
We need it when PEFT version doesn't contain the required prefix.

Reply:
Got it

> a possible fix for the issue in that function is storing deleted keys to ensure they aren't re-added by other iterations

If this issue reoccurs with other keys before minimum PEFT version is increased this can be applied

Reply:
Check now?